plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -205,6 +208,42 @@ public static void main(String[] args)
         }
     }
 
+    public static final class IcebergMinIoFileMetastoreQueryRunnerMain
+    {
+        private IcebergMinIoFileMetastoreQueryRunnerMain() {}
+
+        public static void main(String[] args)
+                throws Exception
+        {
+            String bucketName = "test-bucket";
+            @SuppressWarnings("resource")
+            Minio minio = Minio.builder().build();
+            minio.start();
+            minio.createBucket(bucketName);
+
+            @SuppressWarnings("resource")
+            DistributedQueryRunner queryRunner = IcebergQueryRunner.builder()
+                    .setCoordinatorProperties(Map.of(
+                            "http-server.http.port", "8080"))
+                    .setIcebergProperties(Map.of(
+                            "hive.s3.aws-access-key", MINIO_ACCESS_KEY,
+                            "hive.s3.aws-secret-key", MINIO_SECRET_KEY,
+                            "hive.s3.endpoint", minio.getMinioAddress(),
+                            "hive.s3.path-style-access", "true"))
```

Comment:
NIT: Just wondering if it's not useful to add `.put("hive.s3.streaming.part-size", "5MB") ` here as well

Reply:
i consciously removed it from here. 
this is not tests, so a comment like "reduce tests memory footprint" wouldn't apply.

it makes sense to have same setup as in tests, but they may drift away anyway, so i removed part size from here.

Reply:
btw good catch, i am impressed you noticed that